tests/models/bart/test_modeling_tf_bart.py

```diff
@@ -228,6 +235,77 @@ def test_saved_model_creation(self):
     def test_onnx_compliancy(self):
         pass
 
+    def prepare_config_and_inputs_for_common(self):
+        input_ids = tf.clip_by_value(
+            ids_tensor([self.batch_size, self.seq_length - 1], self.vocab_size),
+            clip_value_min=self.eos_token_id + 1,
+            clip_value_max=self.vocab_size + 1,
+        )
+        eos_tensor = tf.expand_dims(tf.constant([self.eos_token_id] * self.batch_size), 1)
+        input_ids = tf.concat([input_ids, eos_tensor], axis=1)
+
+    # TFBartForSequenceClassification does not support inputs_embeds
```

Comment:
There is no reason to skip this test for TFBartForSequenceClassification. Why doesn't it support inputs_embeds when the base model does?

Reply:
I also asked myself, but found the same test is skipped in Bart testing file for `BartForSequenceClassification`. There is probably some good reason, but it has not been specified at that time.

Reply:
As mentioned above in the clipping, comment, this model relies on access to the raw input_ids to correctly identify the embedding produced for the final `eos` token.   Happy to add a comment in this PR or fast follow with one if it helps.

Reply:
Thanks, it's clear to me now. You are welcome to add a comment regarding the reason to skip, but no obligation ❤️